### PR TITLE
Updated Eatery contributors

### PR DESCRIPTION
**Pull request by JaysonHahn**
## Overview

- Updated Eatery contributors

## Changes Made

- Added the names of new eatery contributors in About Eatery section

## Test Coverage

Used Manual testing. All changes are visible in Profile/Settings icon/About Eatery

## Screenshots (delete if not applicable)

<details>

  <summary>Screen Shot Name</summary>
About Eatery page

  
Simulator Screenshot - iPhone 15 Pro - 2023-09-26 at 00 47 51

</details>

### CODE REVIEW

**Review by vinnie4k — COMMENTED** [review]
Have you confirmed these names with Matt? Eatery Blue has been worked on for many many semesters and there could be names that we are missing so I would double check with Matt.

Also, next time create this branch off of your previous one. Because this PR is small and separate from your changes made in #42, it's fine and you won't get any merge conflicts, but please keep this in mind when you create multiple open PRs. Alternatively, you can merge your previous PR to `main`, then create this branch afterwards off of `main`. Of course, if you are still waiting on reviews then this wouldn't be a good approach but if the other PR has been approved, I recommend merging it first before creating this separate branch.

I will approve this PR once your other PR has been merged.

**Review by vinnie4k — APPROVED** [review]
Good to go now.

**Review by tiffany-pann — APPROVED** [review]
lgtm as long as you've confirmed w matt

**Pull request merged**
